themes/wilson_theme_starterkit/src/js/components/adaptive-panels.js

```diff
@@ -25,11 +25,15 @@
         const panels = accordion.querySelectorAll(".panels__panel");
         const tabs = [];
 
-        // Set up a container that may be used to display tabs.
-        const tabsContainer = document.createElement("ul");
+        // Set up a container that will be used to display tab list.
```

Comment:
When adding this JS to Wave, I spotted that the tabs don't work at all with screen readers. If you tab to a tab, it'll read out the content of the panel on display, but not the tab itself. I think there were some issues when switching between the tabs too but I don't remember what. 

Have you tried this with Voiceover enabled? Will these changes help out with that?

Reply:
Thanks for the heads up @iamsophiesk. These changes are purely to do with the containers and flexbox/grid updates and doesn't aim to fix the accessibility of the tabs. I'd have to carve out some time to place with voice over and see what can be done, but I think that's the subject of a different PR. If we had a Jira board for this, it would certainly be a different ticket. Do you think we could approve this PR without tackling the tabs and I/someone else can come back to that in the near future?

Reply:
Fair enough, thank you for explaining! I'm happy to approve this (I didn't get round to reviewing the rest of the PR yesterday but can take some time today before lunch).

It would be interesting to see @rob-howells input on this as I'm sure there's an easier way to do the tabs anyway - possibly without JS? I feel like we could simplify this whole thing. But yes, definitely one for another time :)

Reply:
Oh, I guess I should copy these changes into Wave, too.

Reply:
The "adaptive panels" code probably could be improved somewhat (my original idea probably isn't very optimised) but I'm not sure we could completely remove the need for Javascript. Not unless we used the Twig templates to inject duplicate headings for when the UI snaps between tabs and accordion (at different breakpoints). Anyway, I've added "Improve screenreader accessibility on Tabs/Accordion component" to wishlist so we can refine and optimise when that gets worked.